recipe_scrapers/nihhealthyeating.py

```diff
@@ -86,3 +86,38 @@ def instructions(self):
         return "\n".join(
             [normalize_string(instruction.get_text()) for instruction in instructions]
         )
+
+    def nutrients(self):
+        res = []
+        nutrition = []
+        
+        for s in self.soup.find('div', {
+            'id': 'nutrition_info'
+        }).find('table').find_all('tr'):
+            for element in s.findChildren('td'):
```

Comment:
```suggestion
            for element in s.find_all('td'):
```

(purely for consistency: `find_all`, `findAll` and `findChildren` are equivalent, as I understand it)